doc/dev/index.rst

```diff
@@ -179,6 +179,100 @@ with::
 
 For more ``vstart.sh`` examples, see :doc:`/dev/quick_guide`.
 
+What is merged where and when ?
+===============================
+
+Commits are merged into branches according to criterions that change
+during the lifecycle of a Ceph release. This chapter is the inventory
+of what can be merged in which branch at a given point in time.
+
+Development releases (i.e. x.0.z)
+---------------------------------
+
+What ?
+^^^^^^
+
+ * features and bug fixes
+
+Where ?
+^^^^^^^
+
+Features are merged to the master branch. Bug fixes should be (but it is not mandatory)
+merged in the branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.).
```

Comment:
Mentioning 0.94.x and 9.2.x here is confusing because they are stable releases, and this section is about Development releases (`x.0.z`). This should say:

Features are merged to the master branch. Bug fixes should be merged to the corresponding named branch (e.g. "jewel" for `10.0.z`, "kraken" for `11.0.z`, etc. However, this is not mandatory - bug fixes can be merged to the master branch as well, since the master branch is periodically merged to the named branch during the development releases phase.